modules/rosa-hcp-aws-private-security-groups.adoc

```diff
@@ -41,20 +47,26 @@ hcp-private
 ----
 $ read -r VPCE_ID VPC_ID <<< $(aws ec2 describe-vpc-endpoints --filters "Name=tag:api.openshift.com/id,Values=$(rosa describe cluster -c ${CLUSTER_NAME} -o yaml | grep '^id: ' | cut -d' ' -f2)" --query 'VpcEndpoints[].[VpcEndpointId,VpcId]' --output text)
 ----
-
-. Create your security group by running the following command:
++
+[WARNING]
+====
+Modifying or removing the default AWS PrivateLink endpoint security group is not supported and may result in unexpected behavior.
```

Comment:
IBM style guide: [Use "may" only to indicate permission](https://www.ibm.com/docs/en/ibm-style?topic=word-usage#m)
```suggestion
Modifying or removing the default AWS PrivateLink endpoint security group is not supported and might result in unexpected behavior.
```